templates/web/greenwich/around/intro.html

```diff
@@ -1,6 +1,7 @@
 <h1>[% loc('Report a problem in Royal Greenwich streets or parks') %]</h1>
 <p>
-    Use this for reporting pysical problems that can be fixed like graffiti,
-    street cleaning issues or road defects. To report other kinds of problems
-    such as missed bins use our <a href="#">reporting forms</a>.
+    Use this for reporting pysical problems that can be fixed like graffiti,street cleaning issues or road defects. 
```

Comment:
@rbg-joe thanks for this! There's a couple of minor typos in this line: 
> Use this for reporting **pysical** problems that can be fixed like **graffiti,street** cleaning issues or road defects.

If you can fix those (happy for you to force push the fixed commit) I'll get this merged in!